pytest_openfiles/plugin.py

```diff
@@ -97,14 +98,19 @@ def pytest_runtest_teardown(item, nextitem):
         ignore = False
 
         for ignored in open_files_ignore:
+
+            # Note that we use fnmatch rather than re for simplicity since
+            # we are dealing with file paths. In addition, we used to check
+            # for exact matches, and we need e.g. "astropy.log" to match
+            # only files with that exact name, and not e.g. "astropyxlog"
```

Comment:
"astropyxlog" makes no sense to me. Is it a typo?
```suggestion
            # only files with that exact name, and not e.g. "astropyxlog"
```

Reply:
The point was that if you use regular expressions, ``astropy.log`` suddenly matches any string of the form ``astropy<character>log`` where character can be e.g. ``x``. But maybe there's a clearer way to write it?

Reply:
Ah, I guess I was confused because `re` was never used (as far as I can see in the diff), so the suddenly comparison to `re` also makes little sense here.

Reply:
I've re-written the comment to be (hopefully) clearer!